app/bundles/EmailBundle/Entity/EmailRepository.php

```diff
@@ -530,6 +530,10 @@ public function resetVariants($relatedIds, $date)
      */
     public function upCount($id, $type = 'sent', $increaseBy = 1, $variant = false)
     {
+        if (!$increaseBy) {
```

Comment:
Yes. If all emails failed to send. Instead of putting a check in every place this could potentially be called, I added here to prevent a non-sensical query (sent_count = sent_count + 0).